controls/cis_rhel7.yml

```diff
@@ -395,11 +395,9 @@ controls:
     levels:
     - l1_server
     - l1_workstation
-    status: partial
+    automated: no
```

Comment:
This should be `status` see https://complianceascode.readthedocs.io/en/latest/manual/developer/03_creating_content.html#reporting-status